locales.software/1038-hu.cfos.txt

```diff
@@ -91,14 +91,14 @@ pnet_conn_local_ip = "Helyi IP cím %?";
 pnet_conn_public_ip = "Nyílvános IP cím %?";
 pnet_pubdir_header = "Nyílvános könyvtár";
 pnet_pubdir_subheader = "Az a könyvtár, amelyik tartalmazza a letölthető fájlokat.";
-pnet_pubdir_info = "Kérlek, add meg azt a mappa helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
+pnet_pubdir_info = "Kérlek, add meg annak a mappának a helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
 smtp_header = "SMTP-szerver";
 smtp_subheader = "SMTP-szerver beállítása.";
 dynip_header = "Dinamikus IP-szerver";
 dynip_subheader = "Dinamikus IP-szerver beállítása";
 dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
 dlg_autostart = "%p indítása a Windows-zal együtt";
-dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
+dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
```

Comment:
I admit it is a weak point on my side, as I do not know what %p exactly
will hold, so it is safe to make in inside a bracket, like (%p). In
Hungarian there are different articles, which depend on the first syllable,
for example: "a kutya", "az asztal", which in English "the dog", "the
table". But in Hungarian the article is different as you can see. As I do
not know what %p holds (I guess it is the program-name, but I do knot know
how it looks exactly), it is best to avoid the confusion with the different
article forms, which easily can be grammatically incorrect.


2015. december 6. 14:46 Serj Lavrin írta, <notifications@github.com>:

> In locales.software/1038-hu.cfos.txt
> <https://github.com/cFos/l10n/pull/15#discussion_r46770480>:
>
> >  smtp_header = "SMTP-szerver";
> >  smtp_subheader = "SMTP-szerver beállítása.";
> >  dynip_header = "Dinamikus IP-szerver";
> >  dynip_subheader = "Dinamikus IP-szerver beállítása";
> >  dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
> >  dlg_autostart = "%p indítása a Windows-zal együtt";
> > -dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
> > +dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
>
> I'm not sure that %p should be placed in (). Is there any particular
> reason behind it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cFos/l10n/pull/15/files#r46770480>.
>